vignettes/DataPipeline.Rmd

```diff
@@ -174,16 +160,16 @@ charts <- metrics %>% map(~Visualize_Metric(
   strMetricID = .x
 )) %>% setNames(metrics)
 
-##function not currently working!
-# Report_KRI(lCharts = charts,
-#            dfSummary = dfSummary, 
-#            dfSite = dfSite,
-#            dfStudy = dfStudy,
-#            dfMetrics = dfMetrics,
-#            strOutput = "~/test_report_kri.html")
+#function not currently working!
```

Comment:
Should we mention an issue related to this? I want to make sure this isn't being left in accidentally (vs intentionally).